src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -153,7 +154,9 @@ func applyClientConfig(client *client.Client, cfg *config.Config, deviceID strin
 	} else {
 		fronted.Configure(certs, cfg.Client.MasqueradeSets, filepath.Join(appdir.General("Lantern"), "masquerade_cache"))
 	}
-	logging.Configure(cfg.CloudConfigCA, deviceID, Version, RevisionDate, cfg.BordaReportInterval, cfg.BordaSamplePercentage, cfg.LogglySamplePercentage)
+	configureLoggingOnce.Do(func() {
```

Comment:
Well, the config is the path by which we get them from the command-line (nothing gets command-line stuff directly, it's all mediated by the config).